### ISSUE TITLE
Oban.check_queue function report jobs running even though all jobs have finished

### ISSUE THREAD

**Issue by TobiasLunaris** [sorentwo/oban#825]
### Environment

- Oban Version: oban 2.13.4, oban_pro 0.12.6
- PostgreSQL Version: 14.5
- Elixir & Erlang/OTP Versions (`elixir --version`): Erlang/OTP 24, Elixir 1.14.2 (compiled with Erlang/OTP 23)

### Current Behavior

I have some `Oban.Pro` workers. In order to keep them busy with work I want to insert new jobs into the queue once the queue is empty. To check the status of the queue I use `Oban.check_queue(queue: my_queue)`.

The problem is that the field `running` in the returned map is not cleared after jobs have finished:

```elixir
iex(2)> Oban.check_queue(queue: :my_queue)

%{
  global_limit: nil,
  local_limit: 5,
  name: "Oban",
  node: "my_node",
  paused: false,
  queue: "my_queue",
  rate_limit: nil,
  retry_attempts: 5,
  retry_backoff: 1000,
  running: [430784, 430785, 430786],
  started_at: ~U[2023-01-08 23:37:18.339451Z],
  updated_at: ~U[2023-01-09 00:31:37.209735Z]
}
```

I have confirmed that there are no jobs in the queue beforehand by running `Oban.cancel_all_jobs(Oban.Job)` and checking the web interface (oban_web 2.9.5).

The `running` field is updated at job insertion though. 

### Expected Behavior

I would have expected `Oban.check_queue(queue: :my_queue)` returned map to have an empty list in the `running` field after all jobs in that queue have been completed:

```elixir
%{
  global_limit: nil,
  local_limit: 5,
  name: "Oban",
  node: "my_node",
  paused: false,
  queue: "my_queue",
  rate_limit: nil,
  retry_attempts: 5,
  retry_backoff: 1000,
  running: [],
  started_at: ~U[2023-01-08 23:37:18.339451Z],
  updated_at: ~U[2023-01-09 00:31:37.209735Z]
}
```

**Comment by sorentwo** [comment]
Thanks for the report. Coincidentally, for an largely unrelated reason, this exact issue was fixed with the previous commit: b304381

The `main` branch isn’t currently compatible with the latest Pro release. I’m targeting a bundled release later this month.

**Comment by TobiasLunaris** [comment]
I see. Glad to hear it has already been addressed. Thank you! ^^
